test/linter/test-links.js

```diff
@@ -172,17 +184,20 @@ function processData(filename) {
   );
 
   return errors;
-}
+};
 
 /**
- * @param {Object[]} errors
- * @param {string} actual
- * @param {string|RegExp} regexp
- * @param {(match: RegExpExecArray) => Object} matchHandler
+ * Given a RegEx expression, test the link for errors
+ *
+ * @param {LinkError[]} errors The errors object to push the new errors to
+ * @param {string} actual The link to test
+ * @param {string|RegExp} regexp The regex to test with
+ * @param {(match: Array.<?string>) => object} matchHandler The callback
+ * @returns {void}
  */
-function processLink(errors, actual, regexp, matchHandler) {
+const processLink = (errors, actual, regexp, matchHandler) => {
   const re = new RegExp(regexp, 'g');
-  /** @type {RegExpExecArray} */
+  /** @type {Array.<?string>} */
```

Comment:
```suggestion
  /** @type {RegExpExecArray} */
```

Reply:
I have not found any documentation, posts, or articles that indicate `RegExpExecArray` is a compatible JSDoc type.